core/migration/relationsfixer.go

```diff
@@ -0,0 +1,88 @@
+package migration
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/go-multierror"
+
+	"github.com/anyproto/anytype-heart/core/block/editor/basic"
+	"github.com/anyproto/anytype-heart/core/block/editor/smartblock"
+	"github.com/anyproto/anytype-heart/pkg/lib/bundle"
+	"github.com/anyproto/anytype-heart/pkg/lib/database"
+	"github.com/anyproto/anytype-heart/pkg/lib/pb/model"
+	"github.com/anyproto/anytype-heart/util/pbtypes"
+)
+
+type readonlyRelationsFixer struct{}
+
+func (readonlyRelationsFixer) Name() string {
+	return "ReadonlyRelationsFixer"
+}
+
+func (readonlyRelationsFixer) Run(ctx context.Context, store QueryableStore, space DoableViaContext) (toMigrate, migrated int, err error) {
+	spaceId := space.Id()
+	var relations []database.Record
+
+	relations, err = listReadonlyTagAndStatusRelations(store, spaceId)
+	toMigrate = len(relations)
+
+	if err != nil {
+		return toMigrate, 0, fmt.Errorf("failed to list all relations with tag and status format in space %s: %w", spaceId, err)
+	}
+
+	if toMigrate != 0 {
+		log.Infof("space %s contains %d relations of tag and status format with relationReadonlyValue=true", spaceId, toMigrate)
+	}
+
+	for _, r := range relations {
+		var (
+			name = pbtypes.GetString(r.Details, bundle.RelationKeyName.String())
+			uk   = pbtypes.GetString(r.Details, bundle.RelationKeyUniqueKey.String())
+		)
+
+		format, ok := model.RelationFormat_name[int32(pbtypes.GetInt64(r.Details, bundle.RelationKeyRelationFormat.String()))]
+		if !ok {
```

Comment:
I don't think it is possible scenario according to your code

Reply:
Thank you! Removed this check